.github/CODEOWNERS

```diff
@@ -1 +1,4 @@
+* @ClickHouse/core
 
+# notify @ClickHouse/integrations-ecosystem when a new type is added
+src/Core/TypeId.h @ClickHouse/core @ClickHouse/integrations-ecosystem 
```

Comment:
To the best of my knowledge, I'd appreciate it if you could point out the correct place if this one isn't sufficient.

Reply:
> also we have system.data_type_families table:

That would work if we need to detect new types in `master`. I want to catch a new type even early in the dev cycle.

> Is it possible to subscribe for new files in directory, in src/DataTypes/* ? 

@vdimir I don't think CODEOWNERS provides this functionality, but we can implement it via GitHub Actions. @Felixoid I might need your help here as the most experienced GHA person for ClickHouse repo

> Then maybe for src/DataTypes/IDataType.h

From what I see, https://github.com/ClickHouse/ClickHouse/commits/master/src/Core/TypeId.h was changed only twice in the last 6 months, https://github.com/ClickHouse/ClickHouse/commits/master/src/DataTypes/IDataType.h was changed 9 times over the previous 6 months.